include/keepkey/firmware/coins.def

```diff
@@ -31,5 +31,6 @@ X(true, "Electra",      true, "ECA",   true,  33,    true,      10000,  true,  4
 X(true, "Nano",         true, "NANO",  true,  NA,    false,        NA,  false, NA,  true, "\x16" "Nano Signed Message:\n",          true, 0x800000a5,   false,   0,     true,  30,  false, NO_CONTRACT,      false, 0,        false, false, false, false, true, ED25519_BLAKE2B_NANO_STRING, false, "",            false, "",     false, false,  false,        0,        false,       0,            true,  "xrb_")
 X(true, "Groestlcoin",  true, "GRS",   true,  36,    true,     100000,  true,   5,  true, "\x1c" "GroestlCoin Signed Message:\n",   true, 0x80000011,   false,   0,     true,   8,  false, NO_CONTRACT,      true, 76067358,  true, true,   true, false,  true,"secp256k1-groestl",          false, "",            true, "grs",   false, false,  true,  77429938,        true, 78792518,            false, ""    )
 X(true, "GRS Testnet",  true, "tGRS",  true, 111,    true,     100000 , true, 196,  true, "\x1c" "GroestlCoin Signed Message:\n",   true, 0x80000001,   false,   0,     true,   8,  false, NO_CONTRACT,      true, 70617039,  true, true,   true, false,  true,"secp256k1-groestl",          false, "",            true, "tgrs",  false, false,  true,  71979618,        true, 73342198,            false, ""    )
+X(true, "Megacoin",     true, "MEC",   true,  50,    true,    2000000,  true, 196,  true, "\u0019MegaCoin Signed Message:\n",       true, 0x800000d9,   false,   0,     true,   8,  false, NO_CONTRACT,      true, 76067358,  true, false,  true, false,  true, SECP256K1_STRING,            false, "",            false, "",     false, false,  false,        0,        false,       0,            false, ""    )
```

Comment:
```suggestion
X(true, "Megacoin",     true, "MEC",   true,  50,    true,    1000000,  true,   5,  true, "\x19" "MegaCoin Signed Message:\n",      true, 0x800000d9,   false,   0,     true,   8,  false, NO_CONTRACT,      true, 76067358,  true, false,  true, false,  true, SECP256K1_STRING,            false, "",            false, "",     false, false,  true,  77429938,        true, 78792518,            false, ""    )
```

Reply:
thank you.

Reply:
I think you need to hit the commit button on the suggestion... I can't.